channels/us.m3u

```diff
@@ -1279,9 +1279,7 @@ https://dai2.xumo.com/amagi_hls_data_xumo1212A-escapes/CDN/master.m3u8
 http://k3.usastreams.com:1935/etvSD/etvSD/playlist.m3u8
 #EXTINF:-1 tvg-id="ESPN.us" tvg-name="ESPN" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/GhhN8mW.png" group-title="Sports",ESPN (360p)
 http://stream.tvtap.live:8081/live/espnusahd.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="ESPN.us" tvg-name="ESPN" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/GhhN8mW.png" group-title="Sports",ESPN (720p)
-http://stream2.grandma.fit/fs2.m3u8
-#EXTINF:-1 tvg-id="ESPN.us" tvg-name="ESPN" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/GhhN8mW.png" group-title="Sports",ESPN (720p)
+#EXTINF:-1 tvg-id="ESPN.us" tvg-name="ESPN" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/GhhN8mW.png" group-title="Sports",ESPN (60fps)
```

Comment:
You can tag it as a 720p just like the others. Still makes a difference

Reply:
I Thought the 720p and other resolutions flags were for channels the weren't multi-bitrate. For example if HBO stream only could do 240p we put 240p. But if it could do 240,360,and 720p we just put HBO

Reply:
Should we let the resolution checker determine what it's really doing? The problem I was having it is in sync with the other (720p) and showing up first in the list while the one that works doesn't show.

Reply:
If I'm understanding how the resolution checker works, it will skip the channels that's already indicating what the resolution is. So I have been leaving it blank. That way it is checked And we know based on being scanned. Another thing I have noticed is all the channels that didn't scan well also either don't work or so choppy that it is making it easy to notice which channels that have isses.

Reply:
> I Thought the 720p and other resolutions flags were for channels the weren't multi-bitrate. For example if HBO stream only could do 240p we put 240p. But if it could do 240,360,and 720p we just put HBO

It shows the best quality available for adaptive link

Reply:
It would be nice if the 'resolution checker' could do what the broken-links checker do. Send the channels that failed the scan to a PR, so they can be further investigated. They clearly seem to have issues. Each one of them that do not have a resolution at the end of the name, either takes forever to load or so choppy the scan can't be completed, so I guess the check moves on.